Packs/SecureWorks/ReleaseNotes/5_0_3.md

```diff
@@ -0,0 +1,11 @@
+
+#### Integrations
+
+##### TaegisXDR v2
```

Comment:
Please refer to https://xsoar.pan.dev/docs/documentation/release-notes#entity-styling:~:text=Entity%20Styling-,%23,-Command%20names%3A%20%2D%20should
For Xsoar best practice release notes.